lib/audio/client/plugin.js

```diff
@@ -37,12 +37,12 @@ class AudioPlugin extends PluginInterface
             gameManager.audioManager.updateDefaultConfig(defaultAudioConfig);
             this.messagesListener.listenMessages(room, gameManager);
         });
-        this.events.on('reldens.preloadUiScene', async (preloadScene) => {
-            preloadScene.load.html('audio', 'assets/html/ui-audio.html');
-            preloadScene.load.html('audio-category', 'assets/html/ui-audio-category-row.html');
+        this.events.on('reldens.preloadUiScene', async (uiSceneManager) => {
+            uiSceneManager.uiSceneDriver.loadHTML('audio', 'assets/html/ui-audio.html');
```

Comment:
This is another example, instead of "loadHTML", "loadContent" can receive a key, and we can use it for both in the phaser-driver:

```
loadContent(key){
    phaserScene.load.html(key, 'assets/html/'+key+'.html');
}
```

Though originally I've used different "key" and "html-file-name" values, we should standarize the naming so we can call the template by it's key.

Reply:
yes, I'm ok ith that but that's a whole new level of refactoring, if I implement that "key" it will break all the code, right now, won't it?